hathor/p2p/peer.py

```diff
@@ -226,14 +240,19 @@ class UnverifiedPeer:
     id: PeerId
     info: PeerInfo = field(default_factory=PeerInfo)
 
-    def to_json(self) -> dict[str, Any]:
+    def to_json(self, only_ipv4_entrypoints: bool = True) -> dict[str, Any]:
```

Comment:
Why don't we add all entrypoints regardless of what's enabled/disabled?

Reply:
This method was being used to build the JSON that is sent to other peers in `ReadyState.send_peers`.

I needed to avoid sending IPv6 endpoints to peers that can't handle it.

I could also just build the response manually there as an alternative, so I don't have to use this `.to_json` method. What do you think?